manifests/openshift-state-metrics-cluster-role.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I really like how you have added only the necessary permissions in the Clusterrole :)